server/server.go

```diff
@@ -66,8 +66,8 @@ func (ms *O11yServer) Run(ctx context.Context) {
 	r.GET("/healthz", func(c *gin.Context) {
 		c.String(http.StatusOK, "healthy")
 	})
-	r.GET("/api/extension/metrics/applications/:application/groupkinds/:groupkind/rows/:row/graphs/:graph", ms.queryMetrics)
-	r.GET("/api/extension/metrics/applications/:application/groupkinds/:groupkind/dashboards", ms.dashboardConfig)
+	r.GET("/extensions/metrics/applications/:application/groupkinds/:groupkind/rows/:row/graphs/:graph", ms.queryMetrics)
+	r.GET("/extensions/metrics/applications/:application/groupkinds/:groupkind/dashboards", ms.dashboardConfig)
```

Comment:
`/extensions/metrics` must be changed to `/api/applications/`: 

so for example, instead of:
`/extensions/metrics/applications/:application/groupkinds/:groupkind/rows/:row/graphs/:graph`
it should be:
`/api/applications/:application/groupkinds/:groupkind/rows/:row/graphs/:graph`

Reply:
Made the change